### ISSUE TITLE
pyc_wheel only works as root

### ISSUE THREAD

**Issue by jonthl** [karpierz/pyc_wheel#1]
If I try to run pyc_wheel as a standard user., I see the following error:

```
Traceback (most recent call last):
  File "/usr/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/usr/local/lib/python3.7/dist-packages/pyc_wheel/__main__.py", line 6, in <module>
    result = main()
  File "/usr/local/lib/python3.7/dist-packages/pyc_wheel/_pyc_wheel.py", line 196, in main
    with_backup=args.with_backup, quiet=args.quiet)
  File "/usr/local/lib/python3.7/dist-packages/pyc_wheel/_pyc_wheel.py", line 86, in convert_wheel
    rewrite_dist_info(dist_info_path, exclude=exclude)
  File "/usr/local/lib/python3.7/dist-packages/pyc_wheel/_pyc_wheel.py", line 111, in rewrite_dist_info
    with record_path.open("r") as record:
  File "/usr/lib/python3.7/pathlib.py", line 1193, in open
    opener=self._opener)
  File "/usr/lib/python3.7/pathlib.py", line 1046, in _opener
    return self._accessor.open(self, flags, mode)
```

If I run this as root, this problem doesn't occur.

**Comment by kmmbvnr** [comment]
The problem inside the rewrite_dist_info function

```
def rewrite_dist_info(dist_info_path: Path, *, exclude=None):
    """Rewrite the record file with pyc files instead of py files."""

    whl_path = dist_info_path.resolve().parent

    # Rewrite the record file with pyc files instead of py files.

    record_path = dist_info_path/"RECORD"
    record_path.chmod(stat.S_IWUSR)  # only write perms to owner!!

    record_data = []
    with record_path.open("r") as record:  # open for read, WUT??
```

**Comment by karpierz** [comment]
Fixes by #4
Very thanks  @suganyaathimuthu :)
